rest/shared/src/main/scala/io/udash/rest/DefaultRESTConnector.scala

```diff
@@ -13,13 +13,21 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.language.implicitConversions
 
 /** Default implementation of [[io.udash.rest.internal.RESTConnector]] for Udash REST. */
-class DefaultRESTConnector(val host: String, val port: Int, val pathPrefix: String)(implicit val ec: ExecutionContext) extends RESTConnector {
+class DefaultRESTConnector(_protocol: String, val host: String, val port: Int, val pathPrefix: String)(implicit val ec: ExecutionContext) extends RESTConnector {
+
+  private val protocol = _protocol match {
```

Comment:
If you follow my previous suggestion, this mapping will probably be inlined / moved to `send()`.